config/settings.yml

```
@@ -3,6 +3,10 @@ environment:
   label: "Beta"
 support_email: becomingateacher@digital.education.gov.uk
 
+search_ui:
+  # URL of the C# search ui app (search-and-compare-ui)
+  base_url: https://localhost:5000
```

```suggestion
  base_url: https://localhost:3002
```
On the basis of https://github.com/DFE-Digital/find-teacher-training/blob/80539f8950cebc7dca02504c4a59135e0642211b/config/puma.rb#L15

---

this is used to generate the find url so it'll be whatever port find is running on locally

---

:face_with_head_bandage: